src/components/pages/GlobalSettings/General/YourInstallation/YourInstallation.tsx

```diff
@@ -0,0 +1,35 @@
+import {FullWidthSpace} from '@custom-antd';
+
+import {ConfigurationCard} from '@molecules';
+
+import {useApiDetailsField} from '@src/store/apiDetails';
+
+import env from '../../../../../env';
+
+import {YourInstallationContainer} from './YourInstallation.styled';
+import YourInstallationItem from './YourInstallationItem';
+
+const YourInstallation: React.FC = () => {
+  const [data] = useApiDetailsField('data');
```

Comment:
You will be not setting here, so you may use `useApiDetailsPick`:
```suggestion
  const {data} = useApiDetailsPick('data');
```

As it will allow multiple values, like:
```suggestion
  const {data, version} = useApiDetailsPick('data', 'version');
```